packages/lab/src/flag-list/FlagListItem.tsx

```diff
@@ -0,0 +1,152 @@
+import { Error, Flag, Help, Warning } from '@material-ui/icons';
+import { ColorDynamic, useResponsiveValue } from '@superdispatch/ui';
+import { forwardRef, ReactNode } from 'react';
+import styled, { css } from 'styled-components';
+import { Button, ButtonProps } from '../button/Button';
+
+type Variant = 'primary' | 'warning' | 'danger';
+
+const IconContainer = styled.div`
+  padding: 4px;
+  border-radius: 50%;
+  background: ${ColorDynamic.Silver200};
+  color: ${ColorDynamic.Blue300};
+  transition: ${({ theme }) =>
+    theme.transitions.create(['color', 'background-color'])};
+
+  & .MuiSvgIcon-root {
+    font-size: 16px;
+  }
+`;
+
+const Content = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+const EndActions = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  gap: 8px;
+
+  ${({ theme }) => theme.breakpoints.down('xs')} {
+    margin-left: 0;
+    width: 100%;
+  }
+`;
+
+const BannerListItemDanger = css`
+  & ${IconContainer} {
+    background: ${ColorDynamic.Red50};
+    color: ${ColorDynamic.Red300};
+  }
+
+  &:hover {
+    background: ${ColorDynamic.Red50};
+  }
+`;
+
+const BannerListItemStandalone = css`
+  background: ${ColorDynamic.Silver200};
+`;
+
+const BannerListContainer = styled.div<{ variant: Variant }>`
```

Comment:
Lets use transitive props

Reply:
@doniyor2109 Can you provide an example of what you mean?

Reply:
See here

https://jakemccambley.medium.com/transient-props-in-styled-components-3105f16cb91f